customerOrders/main.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/io;
+import ballerina/persist;
+
+type CustomerData record {|
```

Comment:
Let's make this just 'Customer'

Reply:
This cannot be done as there is already a record named `Customer` in the persist client generated and hence would give compilation errors.